ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -545,18 +546,27 @@ aaa_server_groups:
 ```yaml
 aaa_authentication:
   login:
-    default: < group | local | none >
-    serial_console: < group | local | none >
+    default: < group name >  | local | none >
+    serial_console: < group name >  | local | none >
   dot1x:
-    default: < group | local | none >
+    default: < group name > | local | none >
+  policies:
+    on_failure_log: < true | false >
+    on_success_log: < true | false >
+    local:
+      allow_nopassword: < false (default) | true >
```

Comment:
No need to include EOS defaults. Only mention if we have defaults in AVD.

Reply:
Good point, fixed!